functions/__fzf_extract_var_info.fish

```diff
@@ -1,28 +1,18 @@
 # helper function for __fzf_search_shell_variables
 function __fzf_extract_var_info --argument-names variable_name set_show_output --description "Extract and reformat lines pertaining to \$variable_name from \$set_show_output."
-    # Extract only the lines that begin with...
-    # $variable_name: set
-    # ...or...
-    # $variable_name[
-    string match --entire --regex "^\\\$$variable_name(?:: set|\[)" <$set_show_output |
+    # Extract only the lines about the variable, all of which begin with
+    # $variable_name: or $variable_name[
+    string match --regex "^\\\$$variable_name(?::|\[).*" <$set_show_output |
```

Comment:
Now that the output doesn't contain lines saying "var not set in global scope", we don't have to specify "set" in the prefix to filter them out.